### PR TITLE
Cms deid 2014

### PR DESCRIPTION
**Pull request by contactlp**
https://bmi-work.kumc.edu/work/ticket/5205#comment:9

### CODE REVIEW

**Comment by dckc** [comment]
In 86504ca, a separate commit for the table/column names would have made things easier to review.

File: deid/bene_id_mapping.sql [hunk 1]
```diff
@@ -13,105 +14,79 @@ select
 from (
   select /*+ PARALLEL(HHA_OCCURRNCE_CODES,12) */ distinct bene_id from HHA_OCCURRNCE_CODES union
   select /*+ PARALLEL(PDE,12) */ distinct bene_id from PDE union
-  select /*+ PARALLEL(BCARRIER_CLAIMS,12) */ distinct bene_id from BCARRIER_CLAIMS union
+  select /*+ PARALLEL(BCARRIER_CLAIMS_K,12) */ distinct bene_id from BCARRIER_CLAIMS_K union
```
Comment (dckc) [inline deid/bene_id_mapping.sql]: This wouldn't work with 2011-2013 data, would it? I guess that's reasonable. I wonder about a separate branch.... Maybe we should tag the code we used for 2011-2013.

File: deid/bene_id_mapping.sql [hunk 2]
```diff
@@ -127,12 +102,10 @@ create table pmap_parts (
 alter table pmap_parts parallel (degree 12);
 
 insert /*+ APPEND */ into pmap_parts
-select /*+ PARALLEL(bene_id_mapping,12) PARALLEL(msis_person,12) PARALLEL(msis_id_mapping,12) */
-  bmap.bene_id_deid, mmap.msis_id_deid, mper.state_cd, 
+select /*+ PARALLEL(bene_id_mapping,12) */
+  bmap.bene_id_deid, null msis_id_deid, null state_cd,
```
Comment (dckc) [inline deid/bene_id_mapping.sql]: I don't understand this change. What is it for?

File: deid/bene_id_mapping.sql [hunk 3]
```diff
@@ -162,4 +135,4 @@ from pmap_parts
 cross join cms_key_sources cks
 where msis_id_deid is not null and state_cd is not null
 ;
-commit;
+commit;
```
Comment (dckc) [inline deid/bene_id_mapping.sql]: oops?

File: deid/hipaa_dob_shift.sql [hunk 4]
```diff
@@ -48,3 +67,9 @@ select
 from age_months am cross join hipaa
 ;
 commit;
+
+/*
+TODO:
+Does 2011-13 data dates need to be shift as some people 
+age would have cross 89 after 2014-15 data ?
+*/
```
Comment (dckc) [inline deid/hipaa_dob_shift.sql]: oops? no newline at end of file

File: deid/bene_id_mapping.sql (hunk 2 above)
Comment (contactlp) [reply deid/bene_id_mapping.sql]: I ll get rid of that sql statement.

Original SQL statement was being used to generate `patient_num`  if  `bene_id_deid`  does not exist for Medicaid data,
As we don't have Medicaid data, I can get rid off it, but it is optional to get rid of the statement as It does not have side effects besides increasing run time.

Comment (contactlp) [reply deid/bene_id_mapping.sql]: fixed it at 01fbe46

File: deid/bene_id_mapping.sql (hunk 1 above)
Comment (contactlp) [reply deid/bene_id_mapping.sql]: tag has been created. https://github.com/kumc-bmi/grouse/releases/tag/2011-2013_CMS_DEID

**Pull request merged**